tests/Unit/Database/Schema/BlueprintTest.php

```diff
@@ -20,13 +20,14 @@ public function setUp()
 
     /**
      * It gets the index alias.
+     *
      * @test
-     * @covers \DesignMyNight\Elasticsearch\Database\Schema\Blueprint::getAlias
+     * @covers       \DesignMyNight\Elasticsearch\Database\Schema\Blueprint::getAlias
      * @dataProvider get_alias_data_provider
      */
     public function it_gets_the_index_alias(string $expected, $alias = null)
     {
-        if (isset($alias)) {
+        if(isset($alias)) {
```

Comment:
No no, spaces definitely go before the parentheses!

```suggestion
        if (isset($alias)) {
```